src/averageAges.js

```diff
@@ -14,12 +14,17 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const onlyMan = people.filter((person) => person.sex === 'm');
+
+  const normalizeMan = !century ? onlyMan : onlyMan
+    .filter((man) => Math.ceil(man.died / 100) === century);
+
+  const sumAge = normalizeMan
+    .reduce((prev, man) => prev + (man.died - man.born), 0);
```

Comment:
Here and in line 51-52 you write pretty common code. Better create function for it.